### PR TITLE
[INS] Use Acc and Gyro calibration validation to calibrate the 1G of the acceleration

### PR DESCRIPTION
**Pull request by JulioCesarMatias**
INS uses the `isImuReady()` function to calibrate the 1G of the acceleration. If the Acc is not calibrated, the INS should be inactive (which does not happen when checking the Gyro).

### CODE REVIEW

File: src/main/flight/imu.c [hunk 1]
```diff
@@ -666,7 +666,7 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
 
 bool isImuReady(void)
 {
-    return sensors(SENSOR_ACC) && gyroIsCalibrationComplete();
+    return sensors(SENSOR_ACC) && STATE(ACCELEROMETER_CALIBRATED);
```
Comment (DzikuVx) [inline src/main/flight/imu.c]: I think it should be rather extended to `sensors(SENSOR_ACC) && gyroIsCalibrationComplete() && STATE(ACCELEROMETER_CALIBRATED)`

**Review by DzikuVx — CHANGES_REQUESTED** [review]

**Comment by JulioCesarMatias** [comment]
@DzikuVx Can we rename the function from `isImuReady()` to `imuIsHealth()` ?

**Comment by DzikuVx** [comment]
@JulioCesarMatias but this function does not checks `health` only basic status of IMU. So not sure about renaming

**Comment by JulioCesarMatias** [comment]
> ok, no problem, let's leave it the way it is

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**